src/test/tests/rendering/scalable.py

```diff
@@ -80,7 +83,8 @@
         # explicitly open a parallel engine, if possible
         # if it fails, the OpenDatabase will start a serial engine
         import socket
-        if "quartz" in socket.gethostname() or \
+        if "dane" in socket.gethostname() or \
+           "quartz" in socket.gethostname() or \
            "pascal"  in socket.gethostname() or \
            "poodle"  in socket.gethostname() or \
```

Comment:
Should we remove the line with poodle and change the comment above to be replaced poodle with dane? Maybe remove quartz, pascal and syrah? I think they have all been retired.

Reply:
I removed all the old systems.